### PR TITLE
Test the `kyma alpha add` command

### PR DESCRIPTION
**Pull request by Cortey**
**Description**

Changes proposed in this pull request:

- rework command args
- move most of the code from the cmd package to the separated place
- test command code 

**Related issue(s)**

- https://github.com/kyma-project/cli/issues/2083

### CODE REVIEW

**Comment by Cortey** [comment]
Feature was tested using short script
`k3d cluster create cluster-tests

go run main.go alpha add --module api-gateway,application-connector,btp-operator,docker-registry,eventing,istio,keda,nats,serverless,telemetry

go run main.go alpha modules --installed

k3d cluster delete cluster-tests
`
All modules installed successfully except for : transparent proxy, cluster-ip, cap-operator which are logged as separate draft-issue

File: internal/communitymodules/cluster/modules.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+	if len(name) != 1 {
+		for _, version := range rec.Versions {
+			if version.Version == name[1] {
+				fmt.Printf("Version %s found for %s\n", version.Version, rec.Name)
+				return version
+			}
+		}
+	}
+
+	fmt.Printf("Using latest version for for %s\n", rec.Name)
```
Comment (MichalKalke) [inline internal/communitymodules/cluster/modules.go:96]: Double `for`

**Review by MichalKalke — COMMENTED** [review]
Works superb! 
Besides two minor things (2 questions) I have another question:
Why we need to pass args like this: `keda,nats` not like: `keda, nats`? It's because of how `Cobra` works, isn't it?

File: internal/communitymodules/cluster/modules.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+		return err
+	}
+
+	customConfig, err := readCustomConfig(crs)
+	if err != nil {
+		return err
+	}
+
+	for _, rec := range available {
+		versionedName := containsModule(rec.Name, modules) //TODO move splitting to earlier
```
[the remark this answers is not in the document]
Comment (pPrecel) [reply internal/communitymodules/cluster/modules.go:31]: I will add it in next PRs

**Comment by pPrecel** [comment]
> Works superb! Besides two minor things (2 questions) I have another question: Why we need to pass args like this: `keda,nats` not like: `keda, nats`? It's because of how `Cobra` work, isn't it?

Yes, it's cobra's thingy

**Review by MichalKalke — APPROVED** [review]

**Pull request merged**